app/uploaders/avatar_uploader.rb

```diff
@@ -0,0 +1,54 @@
+# encoding: utf-8
+
+class AvatarUploader < CarrierWave::Uploader::Base
+
+  # Include RMagick or MiniMagick support:
+  # include CarrierWave::RMagick
+  include CarrierWave::MiniMagick
+
+  # if Rails.env.production? || Rails.env.development?
+    storage :fog
```

Comment:
You probably want `storage :file` for the test environment (especially if using Travis CI).